src/Microsoft.Data.SqlClient/tests/tools/Microsoft.Data.SqlClient.TestUtilities/Microsoft.Data.SqlClient.TestUtilities.csproj

```diff
@@ -12,10 +12,9 @@
     <Copy SourceFiles="config.default.json" DestinationFiles="config.json" Condition="!Exists('config.json')" />
   </Target>
   <ItemGroup>
-    <Content Include="config.json">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-      <Link>config.json</Link>
-    </Content>
+    <None Update="config.json">
```

Comment:
Config json change can be avoided, pipelines are not liking it.

Reply:
If we don't do this then the test version ends up being restored which presumably works for pipelines but won't locally (for me). 
There is a conditional copy of config.defaults.json in the project but i couldn't ever see if being used in the binlog. I assume the logic should be that msbuild will copy config.json from manualtests if it exists but if it doesn't then it should use the default version? At the moment the default version always wins and it comes from the TestUtilities project which is confusing.

Reply:
Manual Tests project does not need to contain config.json anymore.
Config.json belongs to TestUtilities now. Buildguide references were also updated.

Reply:
In which case I've been trying to solve a problem of my own making. Ooops.